src/prebid.js

```diff
@@ -482,8 +486,7 @@ $$PREBID_GLOBAL$$.clearAuction = function() {
  */
 $$PREBID_GLOBAL$$.requestBids = function ({ bidsBackHandler, timeout, adUnits, adUnitCodes }) {
   if (auctionRunning) {
-    utils.logError('Prebid Error: `$$PREBID_GLOBAL$$.requestBids` was called while a previous auction was' +
-      ' still running. Resubmit this request.');
+    bidRequestQueue.push(() => $$PREBID_GLOBAL$$.requestBids(...arguments));
```

Comment:
This does not sufficiently enclose the instance data of a given request for bids, I'll address this in the next commit.